.circleci/config.yml

```diff
@@ -0,0 +1,110 @@
+version: 2.1
+tagged_build_filters: &tagged_build_filters
+  branches:
+    ignore: /.*/
+  tags:
+    only: /v[0-9]+\.[0-9]+\.[0-9]+/
+test_build_filters: &test_build_filters
+  branches:
+    only: /.*/
+  tags:
+    ignore: /v[0-9]+\.[0-9]+\.[0-9]+/
+jobs:
+  test:
+    docker:
+      - image: circleci/ruby:2.6.5
+    steps:
+      - checkout
+      - run: bundle install
+      - run: rubocop -c .rubocop.yml
+  build:
+    docker:
+      - image: circleci/ruby:2.6.5
+    steps:
+      - checkout
+      - run:
+          name: Check Tagged Push
+          command: |
+            PKG_VERSION=$(grep VERSION lib/logdna/version.rb | cut -d'"' -f2)
+            if [[ "${CIRCLE_TAG}" != "v${PKG_VERSION}" ]]; then
+              echo "There is mismatch:"
+              echo "  TAG_VERSION: ${CIRCLE_TAG}"
+              echo "  PKG_VERSION: v${PKG_VERSION}"
+              exit 1
+            fi
+      - run: gem build logdna.gemspec
+      - persist_to_workspace:
+          root: .
+          paths:
+            - ./logdna-*.gem
```

Comment:
is this correct? the filename wouldn't start with `logdna-`

Reply:
@mdeltito, nice catch! Thanks! Updated!